eea/facetednavigation/widgets/sorting/edit.js

```diff
@@ -21,7 +21,7 @@ FacetedEdit.SortingWidget = function(wid, context){
 FacetedEdit.SortingWidget.prototype = {
   set_default: function(element){
     var value = this.select.val();
-    if(this.reverse.attr('checked')){
+    if(this.reverse.is(':checked')){
```

Comment:
It seems prop is preferred but if it's not an expensive loop, both are fine: http://stackoverflow.com/questions/6340294/which-way-to-test-if-an-element-is-checked-is-better-ischecked-or-prop